docs/ldap-schemas/insee.schema

```diff
@@ -548,6 +548,8 @@ attributetype ( 2.5.4.97 NAME 'organizationIdentifier'
 	SUBSTR caseIgnoreSubstringsMatch
 	SYNTAX 1.3.6.1.4.1.1466.115.121.1.15
 	USAGE userApplications )
+attributeTypes: ( 2.5.5.2 NAME 'otherMail'
+    DESC 'Alternative mail' )
```

Comment:
I'm not a fan of adding yet another custom attribute in the ldap schema. Why not use : `inseeTelephoneNumberCorrespondant` which exists already ?

Reply:
inseeTelephoneNumberCorrespondant is SINGLE-VALUE and is about telephone number but I won't find a better matching attribute. Do I remove the SINGLE-VALUE on inseeTelephoneNumberCorrespondant ?

Reply:
Ok I was talking about inseeMailCorrespondant.  And you can leave it as is, we could allow only one supplemntary email adress at the moment.

This is something we will improve with https://github.com/InseeFrLab/sugoi-api/issues/17